app/src/lib/git/core.ts

```diff
@@ -429,6 +429,7 @@ function getDescriptionForError(error: DugiteError): string | null {
     case DugiteError.ConflictModifyDeletedInBranch:
     case DugiteError.MergeCommitNoMainlineOption:
     case DugiteError.UnsafeDirectory:
+    case DugiteError.PathExistsButNotInRef:
```

Comment:
Currently on beta, there is no use case for this and therefore should not be impactful. It will be useful on dev for multi commit diffing (another pr implementing handling in that scenario to be added later)